kite-tools-parent/kite-tools/src/main/java/org/kitesdk/cli/Main.java

```diff
@@ -99,6 +100,7 @@
     jc.addCommand("csv-schema", new CSVSchemaCommand(console));
     jc.addCommand("csv-import", new CSVImportCommand(console));
     jc.addCommand("obj-schema", new ObjectSchemaCommand(console));
+    jc.addCommand("informat-import", new InputFormatImportCommand(console));
```

Comment:
What about `raw-import` or `generic-import`? I think the fact that we're using InputFormat as the API for read "raw" data doesn't mean it has to be in the name of the command.